lib/components/load_spinner/load_spinner.css

```diff
@@ -0,0 +1,55 @@
+@charset "utf-8";
+
+.load-spinner {
+  margin-top: var(--spacer);
+  margin-left: auto;
+  margin-right: auto;
+  border: var(--spacer) solid var(--grey-light--extra);
+  border-radius: 50%;
+  border-top: var(--spacer) solid var(--highlight);
+  width: var(--spacer-6);
+  height: var(--spacer-6);
+  -webkit-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
+}
+
+@-webkit-keyframes spin {
+  0% { -webkit-transform: rotate(0deg); }
+  100% { -webkit-transform: rotate(360deg); }
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+/**
+ * Adapted from Tobias Ahlin's SpinKit https://github.com/tobiasahlin/SpinKit (MIT License)
```

Comment:
if I remember correctly, this is the spinner used by the kk-explorer, right? The original SpinKit library has multiple types of spinners, and this one was adapted from the wave spinner to look more like a bar chart because of the context - so maybe we can rename the css class to be slightly less generic, let's say something like `bar-wave-spinner`?

Reply:
Ahh this makes sense, fixed!